config/settings-production.yml

```diff
@@ -50,6 +61,14 @@ buffy:
         - 2/seeking-reviewer(s)
     ropensci_approve:
       only: editors
+      external_service:
```

Comment:
I am surprised it is called external_service here but external_call for the basic command https://buffy-ropensci.readthedocs.io/en/latest/responders/basic_command.html

Reply:
Yeah, i hadn't noticed that. Confusing indeed!